.npmignore

```diff
@@ -24,9 +24,9 @@ lighthouse-logger/
 
 # Exclude the CLI smoketests but keep the smoketest runner that is used
 # by downstream projects (e.g. publisher ads).
-lighthouse-cli/test/cli
-lighthouse-cli/test/fixtures
-lighthouse-cli/test/smokehouse/test-definitions
+lighthouse-cli/test/*
+!lighthouse-cli/test/smokehouse/
+lighthouse-cli/test/smokehouse/test-definitions/
```

Comment:
almost the exact same thing, just a little more explicitly what the comment describes, and a little more future proof from accidentally including things added to `lighthouse-cli/test/` in the future. The only difference today is that `lighthouse-cli/test/static-server-test.js` is no longer in our published package (a whole 1.5 KiB savings :)